src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: bold;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.searchForm {
+  font-style: normal;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin: 20px auto;
+  line-height: 22px;
+}
+
+.input {
+  position: relative;
+  padding: 25px 62px;
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  font-style: normal;
+  font-weight: bold;
+  color: #3d4e61;
+  transition: background 0.3s ease;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  line-height: 22px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
font styles should be in `html` selector

Reply:
I think that placeholder has its own set of styles, so I need to use ::placeholder pseudo-element. If I remove font styles from placeholder and leave only in html selector, I will have 4 failed tests/

Reply:
you need to inherit font from html selector